### PR TITLE
Update table populate database

### PR DESCRIPTION
**Pull request by Anulfo**
## Status
**READY**

## Testing

[N/A ] I have added passing tests for all new functionality.

[ N/A] I have confirmed that all test prior to this pull request still PASS.

## Documentation

[ ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
Modify the Populate.sql and Table.sql to hold the tables belonging to the Incidents and populate it.
Update the Readme file to tell the user that has to verify that the foreign key is unchecked.

## Deploy Notes

```sh
git pull --prune
git checkout update-table-populate-database
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Touch a bangazonIncident.db file in data.
2.  Open SQL Lite an open that file.
3. On Edit Pragmas unselect `Foreign Key` checkbox.
4. On Execute SQL create the tables using tables.sql.
5. On Execute SQL populate the tables using populate.sql.
 
## Impacted Files in Application
List general components of the application that this PR will affect:

* /data/populate. sql
/data/tables. sql
/customer-service/README.MD

### CODE REVIEW

**Comment by matthamil** [comment]
LGTM other than it says you have conflicts. Also, I don't think we need the database file necessarily in this PR since everyone can seed their own locally 😃 

Should also modify the SQL files in `Data/` so that this database is not seeded with employees.

File: README.md [hunk 1]
```diff
@@ -24,15 +24,18 @@ export ASPNETCORE_ENVIRONMENT="Development"
...
 ```
 
 On initial installation of the Banagazon CLI application you must set an environment variable to your local database.
 ```Bash
-export BangazonWeb_Db_Path="/path/to/bangazon.db"
+export BangazonWeb_Db_Path="/path/to/bangazon_Incident_DB.db"
 ```
 
-Open your `bangazon.db` file in DB Browser, select the `Edit Pragmas` tab and make sure that the `Foreign Keys` checkbox is checked. Open `Data/tables.sql` from the `Execute SQL` tab of DB Browser and execute the statement. Do the same action with `Data/populate.sql`. In DB Browser, click `Write Changes` to commit your changes to the database.
+Open your `bangazon_Incident_Db.db` file in DB Browser, select the `Edit Pragmas` tab and make sure that the `Foreign Keys` checkbox is unchecked.
```
Comment (daynewright) [inline README.md]: Should be `bangazon_Incident_DB.db` to match above  😃

File: README.md [hunk 2]
```diff
@@ -63,10 +66,13 @@ $env:ASPNETCORE_ENVIRONMENT="Development"
 
 On initial installation of the Banagazon CLI application you must set an environment variable to your local database.
 ```Bash
-$env:BangazonWeb_Db_Path="/path/to/bangazon.db"
+$env:BangazonWeb_Db_Path="/path/to/bangazon_Incident_Db.db"
```
Comment (daynewright) [inline README.md]: `bangazon_Incident_DB.db` 😺

File: README.md [hunk 3]
```diff
@@ -63,10 +66,13 @@ $env:ASPNETCORE_ENVIRONMENT="Development"
...
 ```
 
-Open your `bangazon.db` file in DB Browser, select the `Edit Pragmas` tab and make sure that the `Foreign Keys` checkbox is checked. Open `Data/tables.sql` from the `Execute SQL` tab of DB Browser and execute the statement. Do the same action with `Data/populate.sql`. In DB Browser, click `Write Changes` to commit your changes to the database.
+Open your `bangazon_Incident_Db.db` file in DB Browser, select the `Edit Pragmas` tab and make sure that the `Foreign Keys` checkbox is unchecked.
```
Comment (daynewright) [inline README.md]: `bangazon_Incident_DB.db` 😸

**Review by daynewright — CHANGES_REQUESTED** [review]
Readme changes.  Just need consistency with the naming.

File: src/customer-service/Data/tables.sql [hunk 4]
```diff
@@ -68,4 +33,4 @@ CREATE TABLE "Incident"
 	FOREIGN KEY ("IncidentTypeId") REFERENCES "IncidentType" ("IncidentTypeId"),
 	FOREIGN KEY ("OrderId") REFERENCES "Order" ("OrderId"),
```
Comment (MattKraatz) [inline src/customer-service/Data/tables.sql]: This table doesn't exist in the current context, need to drop this FOREIGN KEY statement

File: src/customer-service/Data/tables.sql [hunk 5]
```diff
@@ -68,4 +33,4 @@ CREATE TABLE "Incident"
...
 	FOREIGN KEY ("EmployeeId") REFERENCES "Employee" ("EmployeeId")
```
Comment (MattKraatz) [inline src/customer-service/Data/tables.sql]: Same deal here

**Review by MattKraatz — APPROVED** [review]
LGTM after a quick change to the table.sql file

**Comment by garrett-vangilder** [comment]
LGTM after those changes are made

**Review by daynewright — APPROVED** [review]

**Pull request merged**